src/types/block/redstone.rs

```diff
@@ -121,6 +128,11 @@ pub enum UpdateDirection {
 }
 
 pub mod utils {
+    use super::UpdateDirection;
+
+    pub const DUST_UPDATE_DIRECTION: UpdateDirection = UpdateDirection::FromSource;
+    pub const RAIL_UPDATE_DIRECTION: UpdateDirection = UpdateDirection::AwaySource;
+

```

Comment:
possibly find a better way/place to put this. mby a functions like `pub fn get_update_direction() -> Option<UpdateDirection>`